aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts

```diff
@@ -8,6 +8,7 @@ import { CurveEnum } from 'utils/d3';
 import { SmoothingAlgorithmEnum } from 'utils/smoothingData';
 import { IMetric } from './metricModel';
 import { IRun } from './runModel';
+import { IActivePoint } from '../../../utils/d3/drawHoverAttributes';
```

Comment:
About absolute path also here

Reply:
In some cases this absolute path does not works well and you cant write all paths absolute, i think we can rethink about this